### PR TITLE
Python 3.3 has now reached end-of-life

### PR DESCRIPTION
**Pull request by hugovk**
Remove italics and update the final release to 3.3.7

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by hugovk** [comment]
I've now signed the CLA and now waiting at least one US business day for it to be checked.

File: index.rst [hunk 1]
```diff
@@ -102,8 +102,8 @@ Status of Python branches
 | 3.4              | :pep:`429`   | security    | 2014-03-16     | *2019-03-16*   | `Most recent security release: Python 3.4.7                                |
 |                  |              |             |                |                | <https://www.python.org/downloads/release/python-347/>`_                   |
 +------------------+--------------+-------------+----------------+----------------+----------------------------------------------------------------------------+
-| 3.3              | :pep:`398`   | security    | 2012-09-29     | *2017-09-29*   | `Most recent security release: Python 3.3.6                                |
-|                  |              |             |                |                | <https://www.python.org/downloads/release/python-336/>`_                   |
+| 3.3              | :pep:`398`   | security    | 2012-09-29     | 2017-09-29     | `Final release: Python 3.3.7                                               |
```
Comment (Mariatta) [inline index.rst]: Hmm.. the third column should be end-of-life instead of security?

Comment (hugovk) [reply index.rst]: Good spot, updated!

**Comment by Mariatta** [comment]
Thanks @hugovk and congrats on your first contribution to the devguide :taco:

**Pull request merged**